Classes/Console/Command/Backend/LockBackendForEditorsCommand.php

```diff
@@ -37,7 +37,8 @@ protected function configure(): void
     {
         $this->setDescription('Lock backend for editors');
         $this->setHelp('Deny backend access, but only for editors.
-Admins will still be able to log in and work with the backend.');
+Admins will still be able to log in and work with the backend.
+CLI executions are allowed as well (e.g. a cronjob can execute the TYPO3 scheduler)');
```

Comment:
```suggestion
Admins will still be able to log in and work with the backend. CLI access (commands/Scheduler) is allowed as well');
```

Also please merge the two commits into one and make it a `[TASK]`, see TYPO3 CGL.